message/messaging_service.cc

```diff
@@ -491,7 +491,16 @@ future<> messaging_service::stop_client() {
 
 future<> messaging_service::shutdown() {
     _shutting_down = true;
-    return when_all(stop_nontls_server(), stop_tls_server(), stop_client()).discard_result();
+    auto verbose_stop_action = [] (sstring entity_name,std::function<future<>()> stop_func) -> future<> {
+        mlogger.info("Stopping {}", entity_name);
+        return stop_func().then([entity_name] {
+            mlogger.info("Stopping {} Done", entity_name);
+        });
+    };
```

Comment:
Should be debug level, our logs are already very verbose.

Reply:
I need to check it, but if those logs are debug, maybe we don't need them. I'll verify.

Reply:
It turns out that there is no logging in this area of code.